adds new acid spit genetics mutation obtainable through polysmorphs

iloveloopers opened the pull request:
# Document the changes in your pull request

adds the new acid spit mutation obtainable only from polysmorphs, similarly to fire breath
lets you spit acid that will slowly corrode objects and clothing on players
its weak but annoying
instabillity of 50
cooldown of 18 seconds

# Why is this good for the game?
more species specific mutations is good
the mutation is really only useful for melting stuff, its impractical to use it in combat unless your enemy is naked

# Testing

tested it, works. destroys a common object in 2 shots and clothing depends on the acid armor.

# Wiki Documentation

mutation name: Acid Spit
description: "An ancient mutation from xenomorphs that forces the salivary glands to produce acid"
indicators: none
message: "Your saliva burns your mouth!"
how to obtain: polysmorph species
imcompatible with: none
instabillity: 50
compatible chromosomes: reinforcement, stabilizier

# Changelog

:cl:  
rscadd: New Acid Spit mutation obtainable from Polysmorphs
soundadd: New Acid Spit sound effect
/:cl:

iloveloopers committed:
Add files via upload

iloveloopers committed:
Update DNA.dm

iloveloopers committed:
Update polysmorphs.dm

iloveloopers committed:
Update dna_injector.dm

iloveloopers committed:
Update dna_injector.dm

iloveloopers committed:
Update fire_breath.dm

iloveloopers committed:
18 seconds is fairer

iloveloopers committed:
Update neurotoxin.dm

iloveloopers committed:
Merge branch 'yogstation13:master' into polyspit

iloveloopers committed:
Update fire_breath.dm

iloveloopers committed:
Update code/modules/projectiles/projectile/special/neurotoxin.dm

Co-authored-by: Molti <108117184+Moltijoe@users.noreply.github.com>

iloveloopers committed:
bad molti code fixed!!!

Moltijoe reviewed (changes requested):
undocumented sound addition

iloveloopers committed:
Update fire_breath.dm

iloveloopers committed:
Add files via upload

iloveloopers committed:
Update neurotoxin.dm

iloveloopers committed:
Update acid_spit.dm

iloveloopers committed:
Update yogstation.dme

Moltijoe reviewed (approved):
good enough

iloveloopers committed:
Update acid_spit.dm

iloveloopers committed:
Update acid_spit.dm

iloveloopers committed:
will just not fire at all if your mouth is covered

too complicated to set up specific scenarios on how a breath mask or hardsuit helmet or gas mask or whatever should react differently from being acided

iloveloopers committed:
isn't the end result the same?

Co-authored-by: Molti <108117184+Moltijoe@users.noreply.github.com>

iloveloopers committed:
Update code/datums/mutations/acid_spit.dm

Co-authored-by: Molti <108117184+Moltijoe@users.noreply.github.com>

Moltijoe reviewed (changes requested):
please provide a screenshot or video of it working

iloveloopers committed:
oopsies

Co-authored-by: Molti <108117184+Moltijoe@users.noreply.github.com>
